pylintrc

```diff
@@ -34,6 +34,9 @@ load-plugins=
 # multiple time (only on the command line, not in the configuration file where
 # it should appear only once).
 disable=
+    missing-docstring,
```

Comment:
I feel it's more visible to keep this in, but set a threshold to account for how many docstrings are currently missing. Also, it means that new code isn't held to the higher standard, which I'm not a fan of.